app/assets/stylesheets/nav-menu.scss

```diff
@@ -0,0 +1,47 @@
+@import 'config/import';
```

Comment:
This file should be added to ~ line 30 of `app/views/layouts/_styles.html.erb` so that it renders inline.

Reply:
I just added the reference as mentioned in line ~30 but I have a question though, shouldn't this be added on all/most other `if-else` too? Similar to how `"top-bar.css"` is added in 8 different locations in that same file.

For example, at the current location it will be added inline for pages that have `@article_index = true`. But this is the CSS of the nav menu and it be presented all throughout the app. Unless I don't fully understand the inline optimization  of `layouts/_styles.html.erb` which is possible 😅

Reply:
Hmm this file doesn't really contain any real styling. It does contain some SCSS variables and mixins that are referenced elsewhere (it is also required when you're setting media queries in your CSS because it contains our variables for breakpoints), but I don't think it has to be imported in the `_styles.html.erb`

Reply:
It seems like it's already added as line 37 of `app/views/layouts/_styles.html.erb` so I'll resolve this.

Reply:
@fdoxyz @ludwiczakpawel 

What I think I can do is bring the content of this file to `top-bar.scss` given their relevance to one another. That way, it's better contained in the right place.